logging/cluster-logging-eventrouter.adoc

```diff
@@ -1,4 +1,4 @@
-:context: cluster-logging-curator
```

Comment:
@kabirbhartiRH FYI - This file contained the wrong context, which I fixed. Otherwise, this isn't related to removing the curator content.